django/thunderstore/permissions/models/visibility.py

```diff
@@ -14,6 +14,18 @@ def create_public(self):
             admin_detail=True,
         )
 
+    def create_private(self):
+        return self.create(
+            public_list=False,
+            public_detail=False,
+            owner_list=True,
+            owner_detail=True,
+            moderator_list=True,
+            moderator_detail=True,
+            admin_list=True,
+            admin_detail=True,
+        )
```

Comment:
I suspect calling this `private` could lead to unintended consequences later on (it's not really private as community moderators etc can still see it). I don't necessarily have a good alternative term to suggest but I feel we should reserve `private` for a higher degree of visibility constraints than what's used here. Perhaps something a bit silly like `unpublished`, `nonpublic`, or even `hidden`?